### PR TITLE
Add javascript translation support

### PR DESCRIPTION
**Pull request by piotrrepetowski**
An attempt to solve #759

### CODE REVIEW

File: game/static/game/js/level_moderation.js [hunk 1]
```diff
@@ -47,9 +47,9 @@ var saving = new ocargo.Saving();
 var CONFIRMATION_DATA = {
     'deleteLevel': {
         options: {
-            title: 'Delete level',
+            title: gettext('Delete level'),
         },
-        html: "<p>This student's level will be permanently deleted. Are you sure?</p>",
+        html: '<p>' + gettext("This student's level will be permanently deleted. Are you sure?") + '</p>',
```
Comment (mikebryant) [inline game/static/game/js/level_moderation.js]: In general, there's no need to do this. Our translation software deals with non-text in translations strings fairly well

**Comment by mikebryant** [comment]
Looks good - squash the commits?

File: game/static/game/js/level_moderation.js (hunk 1 above)
Comment (piotrrepetowski) [inline game/static/game/js/level_moderation.js]: You're right. Here I wanted to left only what's required, to avoid mistakes.

**Comment by piotrrepetowski** [comment]
Will do. Before that, one question came to my mind. What do you think about caching the view (as for js-reverse).
Wasn't sure about our approach in this area.

**Comment by CelineBoudier** [comment]
Caching sounds good... This https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#note-on-performance talks about the respective benefits of server-side caching and client-side caching for javascript_catalog view

**Comment by mikebryant** [comment]
Caching sounds good to me - you can use the same technique as I did for `js-reverse`: https://github.com/ocadotechnology/rapid-router/blob/master/game/urls.py#L125

**Comment by piotrrepetowski** [comment]
The cache added. The commits squashed .

**Pull request merged**